sixtyfps_runtime/interpreter/Cargo.toml

```diff
@@ -30,6 +30,7 @@ derive_more = "0.99.5"
 generativity = "1"
 once_cell = "1.5"
 thiserror = "1"
+lazy_static = "1"
```

Comment:
Lazy_static is the old pattern.  The once_cell is the new pattern that might go into the std lib (<https://doc.rust-lang.org/stable/std/lazy/index.html>)
So once_cell should be used instead if possible.

Reply:
Yep, lazy_static is gone again, thanks to your idea about the static ref :)